tests/org.pitest.pitclipse.runner.tests/src/org/pitest/pitclipse/runner/PitRunnerTest.java

```diff
@@ -177,5 +180,38 @@ private static PitOptions options(String classUnderTest, List<String> classesToM
             new File("lib/junit.jar").getAbsolutePath()
         );
     }
-    
+
+    /**
+     * This test case is in place to detect whether the supplied mutators from pit
+     * changed and we need to change {@link Mutators}.
+     */
+    @Test
+    public void testTheMutatorApiOfPit() {
+        assertThat(getPitMutatorsAsString(), equalTo(getExpectedMutatorsAsString()));
```

Comment:
See my single comment on this new test (moving it to a separate test case and using \n)